packages/astro/package.json

```diff
@@ -176,7 +176,8 @@
     "vitefu": "^0.2.5",
     "which-pm": "^2.1.1",
     "yargs-parser": "^21.1.1",
-    "zod": "^3.22.4"
+    "zod": "^3.22.4",
+    "zod-to-json-schema": "^3.22.4"
```

Comment:
The library looks to be actively maintained, and is officially linked from zod's docs. I personally think it is safe to use for now, but we could look into our own implementation in the future.